phpstan-baseline.neon

```diff
@@ -8470,6 +8470,11 @@ parameters:
 			count: 1
 			path: src/lib/REST/Input/Parser/Operation.php
 
+		-
+			message: "#^Cannot call method resolve\\(\\) on Ibexa\\\\Contracts\\\\AdminUi\\\\REST\\\\ApplicationConfigRestResolverInterface\\|null\\.$#"
+			count: 1
+			path: src/lib/REST/Output/ValueObjectVisitor/ApplicationConfigVisitor.php
+
```

Comment:
Added to baseline because Resolver is already checked by `ApplicationConfigRestResolverRegistry::hasResolver` method: https://github.com/ibexa/admin-ui/pull/1027/files#diff-37e803e13a8b6b5d2ef01e755fb2b0591ebde3fe95bbca5869b345c669c730e9R62-R66

Reply:
You can fix it by applying sth like `/** @phpstan-assert-if-true !null $this->resolve() */` on `hasResolver`. See https://phpstan.org/writing-php-code/narrowing-types.

Reply:
Unfortunately this cannot be fixed by `@phpstan-assert-if-true`. This works only for method without arguments. Checked with @Steveb-p.

Reply:
https://phpstan.org/r/50a8757f-9809-4b26-a508-99e9fce04bcb

PHPStan seems unable to recognize that the same arguments are used for both calls, and type narrowing does not happen.

We might try to report it as a missing feature in PHPStan.